perf/perf_nest_events.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+import platform
+import tempfile
+from avocado import Test
+from avocado import main
+from avocado.utils import cpu, distro, genio, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class nestEvents(Test):
+
+    """
+    Tests nest events
+    Collects all the available events from 'perf list' and
+    executes them using 'perf stat' command
+    """
+    # Initializing fail command list
+    fail_cmd = list()
+
+    def setUp(self):
+        '''
+        Install the basic packages to support perf
+        '''
+
+        # Check for basic utilities
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        self.distro_name = detected_distro.name
+        self.cpu_arch = cpu.get_cpu_arch().lower()
+
+        if detected_distro.arch != 'ppc64le':
+            self.cancel('This test is not supported on %s architecture'
+                        % detected_distro.arch)
+        if self.cpu_arch == 'power8':
```

Comment:
avoid extra variable please use this way 
`if cpu.get_cpu_arch().lower() == 'power8':`

Reply:
this as well